workers/unity/Packages/io.improbable.gdk.core/Subscriptions/Reader.cs

```diff
@@ -35,7 +37,7 @@ public TComponent Data
             }
         }
 
-        public Authority Authority
+        public bool Authority
```

Comment:
If we are changing this to `bool` we should rename it to `IsAuthoritative`

Reply:
`HasAuthority` ?